Makefile

```diff
@@ -29,7 +29,7 @@ build_info_ld_flags := $(foreach entry,$(build_info_fields), -X github.com/Kong/
 LD_FLAGS := -ldflags="-s -w $(build_info_ld_flags)"
 GOOS := $(shell go env GOOS)
 GOARCH := $(shell go env GOARCH)
-GOFLAGS := -mod=mod
+GOFLAGS :=
```

Comment:
Probably we don't need this variable anymore

Reply:
Well you never know. Let's keep it as a generic option that marks the places where it is used, so we can quickly add options if we need them in Go 1.15 for example ;)